documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -508,11 +498,11 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 **Description**: Verifies whether operation supports "application/json" as consumes or produces section.
 
-**Why the rule is important**: N/A.
+**Why the rule is important**: Per the ARM SDK guidelines, the AutoRest is used to generated the language SDKs. All the supported language generator currently handles "application/json" but some SDKs may support other content-types. This rule is in place to warn the swagger writter that not all the SDKs would work with other content-types.
 
 **How to fix the violation**: N/A.
```

Comment:
I'd rather say, 'Please consult with AutoRest to ensure that the content type desired can be separated'. Floating the idea of a "Suppression mechanism" in the warning messages could be dangerous. 😄